src/org/usfirst/frc/team467/robot/TiltMonitor.java

```diff
@@ -0,0 +1,23 @@
+package org.usfirst.frc.team467.robot;
+
+import org.apache.log4j.Logger;
+
+public class TiltMonitor {
+	private static TiltMonitor instance;
+	private static final Logger LOGGER = Logger.getLogger(TiltMonitor.class);    
+
+
+	public static TiltMonitor getInstance() {
+		if (instance == null) {
+			instance = new TiltMonitor();
+		}
+		return instance;
+	}
+	public void periodic() {
+		if (Math.abs(Gyrometer.getInstance().getPitchDegrees()) > 45) {
```

Comment:
You should use separate numbers for forward and backwards tilt - the robot may be more tippy forward or backward.

You should put move angles into robot map. Call them FORWARD_PANIC_ANGLE and BACKWARD_PANIC_ANGLE. You can set them both to 45 to start.